packages/@vue/cli-service/lib/commands/serve.js

```diff
@@ -149,10 +152,16 @@ module.exports = (api, options) => {
           return
         }
 
+        let copied = ''
+        if (isFirstCompile && (args.copy || projectDevServerOptions.copy)) {
```

Comment:
As @jkzing suggested I think we should keep it a command line flag only. `--open` is supported in `devServer` because `webpack-dev-server` also supports it.

Reply:
Fixed.